ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -243,120 +243,102 @@
   </Form::Field::ListWrapper>
 
   {{! Account Claim Maps }}
-  <Rose::Form::Fieldset>
-    <:title>{{t 'form.account_claim_maps.label'}}</:title>
-    <:description>{{t 'form.account_claim_maps.help'}}</:description>
-    <:body>
-      {{#if @model.errors.account_claim_maps}}
-        <Rose::Form::Errors as |errors|>
-          {{#each @model.errors.account_claim_maps as |error|}}
-            <errors.message>{{error.message}}</errors.message>
-          {{/each}}
-        </Rose::Form::Errors>
-      {{/if}}
-      <Rose::Table @style='condensed' as |table|>
-        <table.header as |header|>
-          <header.row as |row|>
-            <row.headerCell>{{t 'form.from_claim.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.to_claim.label'}}</row.headerCell>
-            <row.headerCell>{{t 'titles.actions'}}</row.headerCell>
-          </header.row>
-        </table.header>
-        <table.body as |body|>
+  <Hds::Form::Fieldset @isOptional={{true}} as |F|>
+    <F.Legend>{{t 'form.account_claim_maps.label'}}</F.Legend>
+    <F.HelperText>{{t 'form.account_claim_maps.help'}}</F.HelperText>
+    <F.Control>
+      <Hds::Table class='list-wrapper-field'>
+        <:head as |H|>
+          <H.Tr>
+            <H.Th id='fromClaimHeader'>{{t 'form.from_claim.label'}}</H.Th>
+            <H.Th id='toClaimHeader'>{{t 'form.to_claim.label'}}</H.Th>
+            <H.Th>{{t 'titles.actions'}}</H.Th>
+          </H.Tr>
+        </:head>
+        <:body as |B|>
           {{#each @model.account_claim_maps as |claimMap i|}}
-            <body.row as |row|>
-              <row.cell>
-                <form.input
-                  @type='text'
-                  @name='from_claim'
+            <B.Tr>
+              <B.Td>
+                <Hds::Form::TextInput::Field
                   @value={{claimMap.from}}
-                  @error={{@model.errors.account_claim_maps}}
-                  @contextual={{true}}
-                  as |field|
-                >
-                  <field.label>{{t 'form.from_claim.label'}}</field.label>
-                  <field.field />
-                </form.input>
-              </row.cell>
-              <row.cell>
-
+                  @type='text'
```

Comment:
As we discussed in the pass tech time (Thursday 22 February) and as we have being doing with all the `HDS::Text:inputs` for consistency, if the `type=text` we do not declare it since it's the default.

Reply:
@calcaide indeed cleaning up in next commit